src/WritableMemoryStream.ts

```diff
@@ -16,7 +16,7 @@ export default class WritableMemoryStream extends Writable {
     super();
 
     Writable.call(this, options);
-    this.memStore = [Buffer.from('')];
+    this.memStore = [];
```

Comment:
push was failing for me without initializing, can you please check

Reply:
Verified that it is working.